components/FilesTable/FilesTable.vue

```diff
@@ -151,6 +160,12 @@
           </template>
         </el-table-column>
       </el-table>
+      <osparc-file-viewers-dialog
+        :open="dialogSelectedFile !== null"
+        :onClose="() => setDialogSelectedFile(null)"
```

Comment:
This works, but the Vue way is to pass props down and [emit events up](https://vuejs.org/v2/guide/components-custom-events.html). Not saying this needs to change.

More info: https://medium.com/js-dojo/passing-functions-as-props-an-anti-pattern-in-vue-js-b542fc0cf5d

Reply:
I made the change, please check that is the way to go!

Reply:
@ignapas Sorry it took so long to review but this is looking great now!